### PR TITLE
[Php81] Allow not readonly property on API Platform api resource

### PR DESCRIPTION
**Pull request by alexndlm**

### CODE REVIEW

File: rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\Property\ReadOnlyPropertyRector\Fixture;
+
+use ApiPlatform\Metadata\ApiResource;
+
+#[ApiResource]
```
Comment (samsonasik) [inline rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc]: is `@ApiResource` annotation also exists?

Comment (alexndlm) [reply rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc]: In actual versions - no.

Comment (samsonasik) [reply rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc]: in v2.5, annotation is allowed, so I think annotation can be registered as well

https://api-platform.com/docs/v2.5/core/serialization/

Comment (alexndlm) [reply rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc]: ok, added.

File: src/NodeManipulator/PropertyManipulator.php [hunk 2]
```diff
@@ -39,13 +39,23 @@
...
         'Doctrine\ORM\Mapping\Table',
         'Doctrine\ORM\Mapping\MappedSuperclass',
         'Doctrine\ORM\Mapping\Embeddable',
     ];
 
+    /**
+     * @var string[]|class-string<Table>[]
+     */
+    private const ALLOWED_NOT_READONLY_CLASS_ATTRIBUTES = [
+        'ApiPlatform\Metadata\ApiResource',
```
Comment (samsonasik) [inline src/NodeManipulator/PropertyManipulator.php]: this `ApiPlatform\Metadata\ApiResource`  duplicated with already registered on constant `ALLOWED_NOT_READONLY_CLASS_ANNOTATIONS`, I think single constant is fine for both annotation and attribute verification.

Comment (alexndlm) [reply src/NodeManipulator/PropertyManipulator.php]: Oh, thanks. Fixed.

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @alexndlm

**Pull request merged**